### PR TITLE
Menu view

### PR DESCRIPTION
**Pull request by JesperGreen**
PT Story: https://www.pivotaltracker.com/story/show/131070515

Changes proposed in this pull request:
- Added a link on the restaurant page to menu.
- Added menu view/controller/routes
- Refactored to haml
- Refactored basic_steps

What I have learned working on this feature:
- Learned to get path's with helper methods
- Refactoring from html to haml with the help from erb2haml gem
- Cleaner case statement in basic_steps

Screenshots: 
screen shot 2016-09-27 at 6 03 03 pm

screen shot 2016-09-27 at 6 03 37 pm

Ready for review!

### CODE REVIEW

File: app/views/menu/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="row">
+  <div class="column-md-12">
+    <h1>Menu for X</h1>
+    <p>Dish 1</p>
+  </div>
+</div>
```
Comment (tochman) [inline app/views/menu/index.html.erb]: I would really prefer if you guys made a switch to HAML.

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+end
+
+Then(/^I should be on the "([^"]*)" page$/) do |page|
+  set_goto(page)
+  expect(current_path).to eq @goto
+end
+
+def set_goto(page)
+  @goto = '/restaurant' if page == "restaurant"
+  @goto = '/menu'if page == "menu"
```
Comment (tochman) [inline features/step_definitions/basic_steps.rb]: Please use a `case` statement here

Comment (tochman) [reply features/step_definitions/basic_steps.rb]: Also there is no need to use an instance variable, just return a value.

**Review by tochman — CHANGES_REQUESTED** [review]
Please review comments

**Comment by tochman** [comment]
Okay, so for some reason you guys keep having the original pr from @AmberWilkie as the first commit in ALL PR's. With consequential merge conflicts. Please fix that. Make sure you all delete your local `develop` branches and checkout `upstream/develop`

```
$ git checkout -t upstream/develop` 
```

File: app/views/menu/index.html.erb (hunk 1 above)
Comment (JesperGreen) [reply app/views/menu/index.html.erb]: Done

File: features/step_definitions/basic_steps.rb (hunk 2 above)
Comment (JesperGreen) [reply features/step_definitions/basic_steps.rb]: And Done.
Thanks.

**Comment by tochman** [comment]
- I would like you to spend a few minutes and reflect on what you learned while working on this PR. Is it really just the helper methods?
- I would also argue that the HAML translation should have been done in a separate PR with it's own chore in the Tracker. 
- Also, please make use past sentence and bullet points when listing proposed changes (read up on markdown)

**This is not a blocker though so I'll go ahead and merge this. I will look for an updated PR description anyway.**

**Review by tochman — APPROVED** [review]

**Comment by tochman** [comment]
Much better description. Thank you.

**Pull request merged**